server/belga/command/contacts_import.py

```diff
@@ -0,0 +1,151 @@
+                                                            'usage': 'Business',
+                                                            'public': True
+                                                            })
+            if item.get('directPhone2'):
+                doc.setdefault('contact_phone', []).append({'number': item.get('directPhone2'),
+                                                            'usage': 'Business',
+                                                            'public': True
+                                                            })
+            if item.get('personalPhone'):
+                doc.setdefault('contact_phone', []).append({'number': item.get('personalPhone'),
+                                                            'usage': 'Confidential',
+                                                            'public': True
+                                                            })
+            doc['fax'] = ''
+            if item.get('email'):
+                doc.setdefault('contact_email', []).append(item.get('email'))
+            if item.get('personalEmail'):
+                doc.setdefault('contact_email', []).append(item.get('personalEmail'))
+            if item.get('twitter'):
+                doc['twitter'] = item.get('twitter')
+            if item.get('personalTwitter'):
+                doc['twitter_personal'] = item.get('personalTwitter')
+            if item.get('facebook'):
+                doc['facebook'] = item.get('facebook')
+            if item.get('personalFacebook'):
+                doc['facebook_personal'] = item.get('personalFacebook')
+            if item.get('url'):
+                doc['website'] = item.get('url')
+            if item.get('professionalAddress'):
+                doc.setdefault('contact_address', []).append(item.get('professionalAddress'))
+            if item.get('professionalAddress'):
+                doc.setdefault('contact_address', []).append(item.get('personalAddress'))
+            if item.get('Comment1'):
+                doc['notes'] = item.get('Comment1', '')
+            if item.get('keywords'):
+                doc['keywords'] = item.get('keywords')
+            # use original_id check and sync the contact from the belga.
+            doc['original_id'] = str(item.get('contactId'))
+            count_import += 1
+            contact_service.post([doc])
+            docs.append(doc)
+        logger.info("number item: " + str(count_items) + ", number imported item: " + str(count_import))
+        return docs
+
+
+class ContactImportCommand(superdesk.Command):
+    """Import contact from belga to Superdesk.
+    This command use for inserting a large number contact from Belga to Superdesk.
+    Only support for format json file.
+    """
+
+    option_list = [
+        superdesk.Option('--file', '-f', dest='contacts_file_path',
+                         default='/home/thanhnguyen/workspace/0_test/contact/contacts.json')
```

Comment:
I guess there should won't be any default value, or maybe some filename in local folder

Reply:
yes, I forget it. Thanks